scss/login.scss

```diff
@@ -123,6 +123,14 @@ body.modal-open {
       align-items: center;
       justify-content: flex-start;
       flex-direction: column;
+
+      .dc-logo-widget {
+        display: none !important;
```

Comment:
Once I watched a lesson in frontend masters about how to avoid `important`, not sure if it will work but you may try to do this

```css
.dc-logo-widget.dc-logo-widget {
  display: none;
}
```
It could be considered as a hack haha but causes less trouble than !important

![image](https://user-images.githubusercontent.com/20747535/86186306-5cd2cd80-bafe-11ea-85f8-16863ad804c3.png)

Reply:
As a reference I will place the link to the course

https://frontendmasters.com/courses/css-in-depth-v2/

Also, the slides used by the instructor

https://estelle.github.io/cssmastery

Reply:
I was looking at some stylesheets and found this

![image](https://user-images.githubusercontent.com/20747535/86303399-55c8bf80-bbd1-11ea-96ef-8214791df9af.png)

I think this makes this comment resolved